.idea/codeStyles/css-layout.css

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="GreenText" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">#00FF00</item>
+    </style>
+</resources>
+
+/*
```

Comment:
Discussion does not be long in source code comments, especially CSS.  This clutters up the code and leads to technical debt.  The best place for discussion is in the pull request.